src/weights.jl

```diff
@@ -96,8 +96,8 @@ end
 Construct an `AnalyticWeights` vector with weight values `vs`.
 A precomputed sum may be provided as `wsum`.
 
-Analytic weights describe a non-random relative importance (usually between 0 and 1)
-for each observation. These weights may also be referred to as reliability weights,
+Analytic weights represent the inverse of the variance for each case.
```

Comment:
`inverse of the variance` feels kinda redundant here since we also mention that they could also be called inverse variance weights.

Reply:
Yeah maybe we can avoid repeating it later. But I think this should be mentioned here as the definition of the weights.